lib/emailValidator.ts

```diff
@@ -0,0 +1,13 @@
+export const validateEmail = (email: string, options: { style: string } = { style: 'basic' }): boolean => {
+	const basicEmailRegex = /^.+@.+$/;
```

Comment:
looking into some regex you modified to use this function I found this one interesting: `[^@].*@[^@]`, it makes sure only a single `@` is allowed.. wdyt? 👀

Reply:
![image](https://user-images.githubusercontent.com/11577696/144098400-e5e24684-e66f-4d5b-a27f-ebd20568528a.png)
Not sure 🙈 if we should use it tho

Reply:
oopsie.. looks like I missread the regex 🙈

Reply:
ok, this one 😬  `^[^@]+@[^@]+$`